CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/ImportExportPluginsHelpProvider.java

```diff
@@ -45,9 +45,19 @@ public Map<String, String> getHelpMap() {
         final String importExportModulePath = ".." + sep + "ext" + sep + "docs" + sep + "CoreImportExportPlugins" + sep + "src" + sep + "au" + sep + "gov"
                 + sep + "asd" + sep + "tac" + sep + CODEBASE_NAME + sep + "plugins" + sep + "importexport" + sep;
 
-        map.put("au.gov.asd.tac.constellation.plugins.importexport.delimited.DelimitedImportPane", importExportModulePath + "import-from-file.md");
-        map.put("au.gov.asd.tac.constellation.plugins.importexport.export", importExportModulePath + "export-from-constellation.md");
+        //Import Plugins
+        map.put("au.gov.asd.tac.constellation.plugins.importexport.delimited.DelimitedImportPane", importExportModulePath + "import-from-file.md");        
         map.put("au.gov.asd.tac.constellation.plugins.importexport.jdbc.JDBCImportPane", importExportModulePath + "import-from-database.md");
+        
+        //Export Plugins
+        map.put("au.gov.asd.tac.constellation.plugins.importexport.export", importExportModulePath + "export-from-constellation.md");
+        map.put("au.gov.asd.tac.constellation.plugins.importexport.geospatial.ExportToGeoJsonPlugin", importExportModulePath + "export-from-constellation.md");
+        map.put("au.gov.asd.tac.constellation.plugins.importexport.geospatial..ExportToImagePlugin", importExportModulePath + "export-from-constellation.md");
```

Comment:
Currently, the `ExportToImageAction` and the `ExportToJSONAction` create a custom `JFileChooser' whilst the other export plugins execute interactively with a `PluginParametersSwingDialog` before initiating the `JFileChooser`. The help buttons have been activated for the `PluginParametersSwingDialog` but not the `JFileChooser`. From some brief research I cannot find an easy way to activate a help button in the `JFileChooser`. 

To get around this, we can.
- Simply refactor the `ExportToImageAction` and the `ExportToJSONAction` to execute interactively and therefore with a `PluginParametersSwingDialog`, adding another step in the execution process but making them conformant with the other export plugins. 
- Find a way to enable help in the File Chooser, potentially requiring a refactor of the file chooser from swing to a JavaFX alternative.

Anything thoughts or opinions? @Auriga2, @Delphinus8821, @antares1470, @OrionsGuardian

Reply:
Probably not worth the effort then? We can create a new ticket if anyone thinks it's needed.

Reply:
I think it could be worth making a new ticket for changing the File Chooser to JavaFX as it will give us more flexibility in the future but it doesn't need to be done as a part of this ticket.

Reply:
I'm wondering whether export to image or json really need a help button? It's worth noting that for the other export options, they have parameters which are detailed in the help page whereas image and json are simply exporting the graph as a png and json respectively with nothing further to understand.

We could still try things to add a help button for those actions but I suspect the effort required won't be worth the value generated. Making it interactive will add an extra button to click to select the file and jfx alternative probably only worth doing if we then apply to all dialogs for consistency which could be a lot of work for a couple of plugins

Reply:
General consensus is that adding help for these two-export plugins is not going to be included in this ticket. I'll mark this conversation as resolved and we can revisit this topic if conversations around help functionality in dialogue displayers are brought up again in the future.